ui/oopsyraidsy/data/04-sb/dungeon/temple_of_the_fist.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// Temple of the Fist
+[{
+  zoneRegex: /Temple Of The Fist/,
+  damageWarn: {
+    'Temple Fire Break': '21ED', // Conal AoE, Bloodglider Monk trash
+    'Temple Radial Blaster': '1FD3', // Circle AoE, boss 1
+    'Temple Wide Blaster': '1FD4', // Conal AoE, boss 1
+    'Temple Crippling Blow': '2016', // Line AoEs, environmental, before boss 2
+    'Temple Broken Earth': '236E', // Circle AoE, Singha trash
+    'Temple Shear': '1FDD', // Dual conal AoE, boss 2
+    'Temple Counter Parry': '1FE0', // Retaliation for incorrect direction after Killer Instinct, boss 2
+    'Temple Tapas': '', // Tracking circular ground AoEs, boss 2
+    'Temple Hellseal': '200F', // Red/Blue symbol failure, boss 2
+    'Temple Pure Will': '2017', // Circle AoE, Spirit Flame trash, before boss 3
+    'Temple Megablaster': '163', // Conal AoE, Coeurl Prana trash, before boss 3
+    'Temple Windburn': '1FE8', // Circle AoE, Twister wind, boss 3
+    'Temple Hurricane Kick': '1FE5', // 270-degree frontal AoE, boss 3
+    'Temple Silent Roar': '1FEB', // Frontal line AoE, boss 3
+    'Temple Mighty Blow': '1FEA', // Contact with coeurl head, boss 3
+  },
+  triggers: [
+    {
+      id: 'Temple Heat Lightning', // Purple spread circles, boss 1
+      damageRegex: '1FD7',
+      condition: function(e, data) {
+        // Double taps only
+        return e.type != '15';
+      },
+      mistake: function(e, data) {
+        return { type: 'warn', blame: e.targetName, text: e.abilityName };
+      },
```

Comment:
I don't think the `data` parameters are necessary here.

Reply:
It's possible, but this does follow conventions from elsewhere in the repository. (See [E2s](https://github.com/quisquous/cactbot/blob/26244749fac6e5d6d1ee187e423dc6b9f37ecce1/ui/oopsyraidsy/data/05-shb/raid/e2s.js#L30) or [Titania EX](https://github.com/quisquous/cactbot/blob/26244749fac6e5d6d1ee187e423dc6b9f37ecce1/ui/oopsyraidsy/data/05-shb/trial/titania-ex.js#L32) for examples of Quisquous' work matching this structure.) I'm not attached to it though, if it really is unnecessary I'm happy to leave it out.

Reply:
True; I only mention it because I started noticing the same, but I think they're remnants from copy/paste, since the functionality of just an event without data exists (see [O4S' oopsyraidsy entry](https://github.com/quisquous/cactbot/blob/master/ui/oopsyraidsy/data/04-sb/raid/o4s.js#L44-L46)).

Reply:
I suppose that's true. I'll update that then.